brocade-switches.yaml

```diff
@@ -104,3 +104,26 @@
         state: present
       delegate_to: 127.0.0.1
       loop: "{{ range(49, 52 + 1, 1)|list }}"
+
+    - name: "Create management interface of the switch"
+      netbox.netbox.netbox_device_interface:
+        netbox_url: "{{ netbox_url }}"
+        netbox_token: "{{ netbox_token }}"
+        data:
+          device: "{{ brocade_hostname }}"
+          name: management
+          type: 1000Base-t (1GE)
+          mgmt_only: true
+        state: present
+      delegate_to: 127.0.0.1
+
+    - name: "Associate IPMI address with IPMI interface"
+      netbox.netbox.netbox_ip_address:
+        netbox_url: "{{ netbox_url }}"
+        netbox_token: "{{ netbox_token }}"
+        data:
+          address: "{{ ansible_ssh_host }}"
```

Comment:
This should be `ansible_host`. See e.g. [this note](https://docs.ansible.com/ansible/2.3/intro_inventory.html):

> Ansible 2.0 has deprecated the “ssh” from ansible_ssh_user, ansible_ssh_host, and ansible_ssh_port to become ansible_user, ansible_host, and ansible_port. If you are using a version of Ansible prior to 2.0, you should continue using the older style variables (ansible_ssh_*). These shorter variables are ignored, without warning, in older versions of Ansible.

Reply:
thanks for pointing it out! fixed it.